### PR TITLE
{CI} Update service team request template

### PR DESCRIPTION
**Pull request by wangzelin007**
**Related command**

**Description**

[New template](https://github.com/Azure/azure-cli/blob/8cd5dcd70d30077a14fa73c14430472d49fa630e/.github/ISSUE_TEMPLATE/Service_team_request.yml)
1. Add PM contact
2. Add engineer contact
image
3. Add checkboxes for preconditions
image

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

**Review by zhoxing-ms — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/Service_team_request.yml [hunk 1]
```diff
@@ -8,6 +8,18 @@ body:
       value: |
         [How to Request a Feature in CLI](https://github.com/Azure/azure-cli/blob/dev/doc/how_to_request_a_feature_in_cli.md)
         [A Great Example](https://github.com/Azure/azure-cli/issues/22275)
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Preconditions
+      description: |
+        If you need to upgrade the api-version, please ensure that the python SDK is ready. 
+        You can refer to the above documents to release a new version of the python SDK:
```
Comment (zhoxing-ms) [inline .github/ISSUE_TEMPLATE/Service_team_request.yml:17]:
```suggestion
        If it is related to a command module that relies on the Python SDK *(the module has not been migrated to Code Gen yet)* and requires upgrading the api-version
, please ensure that the Python SDK is ready. 
        You can refer to the above documents to release a new version of the Python SDK:
```

File: .github/ISSUE_TEMPLATE/Service_team_request.yml [hunk 2]
```diff
@@ -8,6 +8,18 @@ body:
...
+        - https://github.com/Azure/azure-rest-api-specs/wiki/Abstract-about-how-to-ask-for-public-SDK-release
+        - https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/release-request/rules-for-release-request.md
+      options:
+        - label: No need to upgrade api-version or the python SDK is ready.
```
Comment (zhoxing-ms) [inline .github/ISSUE_TEMPLATE/Service_team_request.yml:21]:
```suggestion
        - label: No need to upgrade Python SDK or the Python SDK is ready.
```

**Review by zhoxing-ms — APPROVED** [review]

**Comment by jsntcy** [comment]
We can add some words to promote codegen tool to partner teams and apply them to PR template as well.
In CI, we can add a linter rule if the code calls rest api directly, we can add a warning to tell them to use codegen tool instead.

File: .github/ISSUE_TEMPLATE/Service_team_request.yml [hunk 3]
```diff
@@ -8,6 +8,18 @@ body:
       value: |
         [How to Request a Feature in CLI](https://github.com/Azure/azure-cli/blob/dev/doc/how_to_request_a_feature_in_cli.md)
         [A Great Example](https://github.com/Azure/azure-cli/issues/22275)
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Preconditions
+      description: |
+        If it is related to a command module that relies on the Python SDK *(the module has not been migrated to Code Gen V2 yet)* and requires upgrading the api-version, please ensure that the Python SDK is ready.
```
Comment (kairu-ms) [inline .github/ISSUE_TEMPLATE/Service_team_request.yml:16]:
```suggestion
        If the commands relies on Python SDK and requires upgrading the api-version, please ensure that the Python SDK is ready. **It's not required for the commands generated by [aaz-dev-tools](https://azure.github.io/aaz-dev-tools/) (Code Gen V2).**
```

**Review by bebound — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/Service_team_request.yml [hunk 4]
```diff
@@ -8,6 +8,26 @@ body:
       value: |
         [How to Request a Feature in CLI](https://github.com/Azure/azure-cli/blob/dev/doc/how_to_request_a_feature_in_cli.md)
         [A Great Example](https://github.com/Azure/azure-cli/issues/22275)
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Preconditions
+      description: |
+        If the commands relies on Python SDK and requires upgrading the api-version, please ensure that the Python SDK is ready. 
+        You can refer to the above documents to release a new version of the Python SDK:
```
Comment (jsntcy) [inline .github/ISSUE_TEMPLATE/Service_team_request.yml:17]:
```suggestion
        You can refer to the documents below to release a new version of the Python SDK:
```

File: .github/ISSUE_TEMPLATE/Service_team_request.yml [hunk 5]
```diff
@@ -73,6 +93,22 @@ body:
       placeholder: e.g., 2022-07-05
     validations:
       required: true
+  - type: input
+    id: pm
+    attributes:
+      label: PM Contact
+      description: Please provide PM contact information or github account.
```
Comment (jsntcy) [inline .github/ISSUE_TEMPLATE/Service_team_request.yml:100]:
```suggestion
      description: Please provide PM contact (Microsoft alias).
```

File: .github/ISSUE_TEMPLATE/Service_team_request.yml [hunk 6]
```diff
@@ -73,6 +93,22 @@ body:
...
+      placeholder: e.g., @pm
+    validations:
+      required: true
+  - type: input
+    id: engineer
+    attributes:
+      label: Engineer Contact
+      description: Please provide Engineer contact information or github account.
```
Comment (jsntcy) [inline .github/ISSUE_TEMPLATE/Service_team_request.yml:108]:
```suggestion
      description: Please provide Engineer contact (Microsoft alias).
```

**Review by jsntcy — APPROVED** [review]

**Pull request merged**